### DISCUSSION TITLE
Is felte still maintained?

### DISCUSSION THREAD

**Discussion by akkie** [pablo-abc/felte#236] [General]
Hi @pablo-abc,

Thanks for your work and for making this library open source. Looking at the history, it seems that felte has not been actively maintained for a few months. Would you currently recommend using it for a new project? I know things change, and I have abandoned open source projects in the past. But I would like to make sure that the project has a future if I am going to use it. Perhaps there is a way for people to support you?

**Comment by pablo-abc** [comment]
I've written this on Reddit as well. I have no plans to stop maintaining this at all (I still use it on personal projects) but I have sadly not have enough time to look at (and solve) the issues other users have found.

It's definitely something I still use and find useful. And I'm, of course, willing to review and merge requests that could help me!
